### PR TITLE
Companies domain check

### PR DESCRIPTION
**Pull request by rwhite27**
Features:

- Add missing features in companies creation
- Format code

### CODE REVIEW

File: src/Utils/Str.php [hunk 1]
```diff
@@ -0,0 +1,248 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Utils;
+
+class Str
```
Comment (kaioken) [inline src/Utils/Str.php:7]: @rwhite27 we dont need this? since laavel hast it

File: routes/api.php [hunk 2]
```diff
@@ -29,4 +29,15 @@
         Route::put('/apps/{id}', 'update');
         Route::delete('/apps/{id}', 'destroy');
     });
+
+    /**
+     * Companies Routes.
+     */
+    Route::group(['controller' => '\Kanvas\Http\Controllers\Companies\CompaniesController'], function () {
```
Comment (kaioken) [inline routes/api.php:36]: @rwhite27 isn't it better to use CompaniesController::class  ?

**Review by kaioken — CHANGES_REQUESTED** [review]

File: routes/api.php (hunk 2 above)
Comment (rwhite27) [reply routes/api.php:36]: maybe. I'll leave the full namespace reference though.

File: src/Utils/Str.php (hunk 1 above)
Comment (rwhite27) [reply src/Utils/Str.php:7]: Maybe we do have some that Laravel doesn't.

Comment (kaioken) [reply src/Utils/Str.php:7]: @rwhite27 we should extend from https://github.com/illuminate/support/blob/master/Str.php and leave the ones we have that they dont

File: src/Utils/Json.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+class Json
+{
+    /**
+     * Given a string determine if its a json.
+     *
+     * @param string|null $string
+     *
+     * @return bool
+     */
+    public static function isJson(?string $string = null) : bool
```
Comment (kaioken) [inline src/Utils/Json.php:18]: @rwhite27 lets use https://github.com/illuminate/support/blob/master/Str.php#L388

**Review by kaioken — CHANGES_REQUESTED** [review]

**Pull request merged**